docs/RaidbossGuide.md

```diff
@@ -63,29 +63,28 @@ Boolean, defaults to true. If true, timelines and triggers will reset automatica
 ```javascript
 {
   id: 'id string',
+  disabled: false,
   regex: /trigger-regex-(with-position-1)(and-position-2)-here/,
   regexFr: /trigger-regex-(with-position-1)-here-but-in-french/,
-  delaySeconds: 0,
-  suppressSeconds: 0,
-  durationSeconds: 3,
   condition: function(data, matches) { return true if it should run },
   preRun: function(data, matches) { do stuff.. },
   promise: function(data, matches) { return promise to wait for resolution of },
-  infoText: {'Info popup'},
-  alertText: {'Alert Popup'},
-  alarmText: {'Alarm Popup'},
-  tts: {'TTS text'},
-  response: {
-    infoText: {'Info popup'},
-    alertText: {'Alert Popup'},
+  delaySeconds: 0,
+  durationSeconds: 3,
+  suppressSeconds: 0,
+  sound: '',
+  soundVolume: 1,
+   response: {
     alarmText: {'Alarm Popup'},
+    alertText: {'Alert Popup'},
+    infoText: {'Info Popup'},
     tts: {'TTS text'},
   },
```

Comment:
I know this was here before but `alarmText: {'Alarm Popup'}` is a bit ambiguous.  Maybe it should be either a string, or a function returning a string, but maybe not an invalid object.

(Also I think response shouldn't be indented?)

Reply:
Looking at this more, do we really want to use a full object for `response` as opposed to a `Responses.foo()` function? Maybe 

```
response: Responses.doSomething(triggerSeverity),
alarmText: {en: 'Alarm Popup'},
alertText: {en: 'Alert Popup'},
infoText: {en: 'Info Popup'},
```
I'm not sure how often it's really going to happen that we return a localized object directly from `response` instead of a function? Or is there a more common use case I'm not thinking of?

(Someday copy/paste will be able to read our minds with respect to indents. Someday!)